airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSource.java

```diff
@@ -566,30 +580,43 @@ private static AirbyteStream addCdcMetadataColumns(final AirbyteStream stream) {
   private void readSsl(final JsonNode sslMethod, final List<String> additionalParameters) {
     final JsonNode config = sslMethod.get("ssl_method");
     switch (config.get("ssl_method").asText()) {
-      case "unencrypted" -> additionalParameters.add("encrypt=false");
+      case "unencrypted" -> {
+        additionalParameters.add("encrypt=false");
+        additionalParameters.add("trustServerCertificate=true");
```

Comment:
Only added for clarity. By default, encrypt=false implies trustServerCertificate=true.

Reply:
trustServerCertificate is trust all essentially. 

This will trust any server certificate. I assume for the exchange of credentials which is still encrypted event when `encrypt=false` (you mentioned this offline)